book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] All these attributes will need to be readable and writable. Remember the `Rental` class interacts with the **`rentals`** attribute of **`Book`** to be able to write it.

```suggestion
  attr_accessor :title, :author, :rentals
```